source/java/product-guide/index.markdown

```diff
@@ -3721,6 +3721,218 @@ The returned `FacebookProviderData` will include:
 
 ***
 
+<a class="anchor" name="integration-github"></a>
+## Integrating with Github
+
+Stormpath supports accessing accounts from a number of different locations including Github.  Github uses OAuth 2.0 protocol for authentication / authorization and Stormpath can leverage their or access tokens to return an `Account` for a given code. 
```

Comment:
Should have been "Stormpath can leverage their access tokens"... Good catch, thanks!